src/Components/Modal/Modal.module.css

```diff
@@ -0,0 +1,138 @@
+/* The Modal (background) */
+.modal {
+  display: none; /* Hidden by default */
+  position: fixed; /* Stay in place */
+  z-index: 1; /* Sit on top */
+  padding-top: 100px; /* Location of the box */
+  left: 0;
+  top: 0;
+  width: 100%; /* Full width */
+  height: 100%; /* Full height */
+  overflow: auto; /* Enable scroll if needed */
+  background-color: rgb(0, 0, 0); /* Fallback color */
+  background-color: rgba(0, 0, 0, 0.4); /* Black w/ opacity */
+}
+
+/* Modal Content */
+.modalContent {
+  position: relative;
+  background-color: var(--lightgrey);
+  margin: auto;
+  padding: 0;
+  border: 1px solid #888;
+  border-radius: 1%;
+  width: 80%;
+  box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
+  -webkit-animation-name: animatetop;
+  -webkit-animation-duration: 0.4s;
+  animation-name: animatetop;
+  animation-duration: 0.4s;
+}
+
+/* Add Animation */
+@-webkit-keyframes animatetop {
+  from {
+    top: -300px;
+    opacity: 0;
+  }
+  to {
+    top: 0;
+    opacity: 1;
+  }
+}
+
+@keyframes animatetop {
+  from {
+    top: -300px;
+    opacity: 0;
+  }
+  to {
+    top: 0;
+    opacity: 1;
+  }
+}
+
+/* The Close Button */
+.close {
+  color: var(--white);
+  float: right;
+  font-size: 25px;
+  width: 40px;
+  height: 40px;
+  text-align: center;
+  background-color: var(--translucent-lightgrey);
+  border-radius: 50%;
+  margin: 10px;
+}
+
+.close:hover,
+.close:focus {
+  color: #000;
+  text-decoration: none;
+  cursor: pointer;
+  background-color: var(--semi-translucent-white);
+}
+
+.modalHeader {
+  padding: 20px 35px;
+  /* background-color: var(--white); */
+  color: var(--canva-blue);
+}
+
+.leftModal {
+  background-color: #fefefe;
+}
+
+.steps {
+  padding: 20px 45px;
+}
+
+.modalBody {
+  height: 700px;
+  background-color: var(--lightgrey);
+  text-align: center;
+}
+
+.createDesignBtn {
+  position: absolute;
+  right: 70px;
+  top: 10px;
+  height: 45px;
+  width: 155px;
+}
+/* laptop screen size */
+@media only screen and (max-width: 1024px) {
```

Comment:
you can omit the `only screen` check. Also a good idea to use variables for the breakpoints (eg. `desktop-breakpoint`), again to keep things consistent/make changes in one place only